src/use_cases/uc_delete.rs

```diff
@@ -1,39 +1,60 @@
 use crate::commutator::Application;
 use crate::storage;
+use actix_web::web::patch;
+use json::object;
+use service::{Context, Services};
 use std::fs::File;
 use std::io::{BufRead, BufReader, Error};
 
 pub fn delete(app: &Application) -> Result<(), Error> {
   let mut count = 0;
-  for ws in app.wss.list()? {
-    for doc in ws.clone().into_iter() {
-      // println!("{:?} {:?}", doc.id, doc.json()?);
+  let ws = app.wss.list()?[0].clone();
```

Comment:
why you think that `[0]` is correct one?

Reply:
Can do let ws = "yjmgJUmDo_kn9uxVi8s9Mj9mgGRJISxRt63wT46NyTQ" for Midas